File: Day1/Find_Maximum_Element/Find_Maximum_Element.py
def find_max (arr): # type: ignore
    max = arr[0] # type: ignore
    for num in arr: # type: ignore
        if num > max:
            max = num  # type: ignore
    return max  # type: ignore

print(find_max([3,4,2,6,9,2,0,-1,5]))

File: Day1/Find_Maximum_Element/Find_Maximum_Element.java
package Day1.Find_Maximum_Element;

class MaxElement {
    public static int findMax(int[] arr) {
        int maxVal = arr[0];
        for(int num : arr) {
            if(num > maxVal) maxVal = num;
        }
        return maxVal;
    }
    public static void main(String[] args) {
        int[] arr = {3,7,2,9,5};
        System.out.println(findMax(arr)); // 9
    }
}
